changes.go

```diff
@@ -7,6 +7,19 @@ import (
 	"net/http"
 )
 
+// RevisionKind describes the change kind.
+//
+// Gerrit API docs: https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#revision-info
+type RevisionKind string
+
+const (
+	Rework                 RevisionKind = "REWORK"
+	TrivialRebase          RevisionKind = "TRIVIAL_REBASE"
+	MergeFirstParentUpdate              = "MERGE_FIRST_PARENT_UPDATE"
+	NoCodeChange                        = "NO_CODE_CHANGE"
+	NoChange                            = "NO_CHANGE"
```

Comment:
unrelated change?

Reply:
or should they also be `RevisionKind`?

Reply:
After running `gofmt -s` these repetitions are gone, I think these are simplifications?

Reply:
Done.